### DISCUSSION TITLE
RCD can't be used on Flockwalls and Flock floortiles

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#218] [Ideas]
Discussion if RCD shouldn't be able to be used on Flockwalls or Flock floortiles.

Would be a property of the Flock that the RCD is incompatible with Flock stuff, similar to how the cargo tele can't teleport Flock closets. Would reinforce the idea that you need to deal damage to these in order to remove them.

On the other hand, could be nice in the same situations that you would need it for regular station tiles, besides the damage part.

**Comment by stonepillars** [comment]
I can see deconstructing with the RCD being sort-of okay since it takes mats and RCD mats can be pretty expensive. But the stock RCD in EVA can wreck alot of havoc against the flock if used correctly. Maybe it'd cost more?
